### PR TITLE
Multi-selection of Edit History items for comparison.

### PR DESCRIPTION
**Pull request by dbrant**
https://phabricator.wikimedia.org/T299180

### CODE REVIEW

File: app/src/main/java/org/wikipedia/page/edithistory/EditHistoryItemView.kt [hunk 1]
```diff
@@ -13,11 +18,30 @@ import org.wikipedia.util.ResourceUtil
...
+        }
+        binding.clickTargetView.setOnLongClickListener {
+            listener?.onLongClick()
+            true
+        }
+        binding.selectButton.setOnClickListener {
+            listener?.onToggleSelect()
+        }
+        binding.userNameText.setOnClickListener {
+            // TODO
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/page/edithistory/EditHistoryItemView.kt:43]: Should we show a menu here?

**Review by cooltey — CHANGES_REQUESTED** [review]
The code looks good to me, but some places may be optimized:
1. When in "compare" mode, the back button should be able to cancel the "compare" mode.
2. Should we make the whole item togglable when in "compare" mode instead of leading the user to the detail view when clicking on the item? Feel like only making the radio button togglable is not enough.
3. According to the design, the first item should be yellow, and the second one should be blue.

File: app/src/main/java/org/wikipedia/page/edithistory/EditHistoryItemView.kt (hunk 1 above)
Comment (dbrant) [reply app/src/main/java/org/wikipedia/page/edithistory/EditHistoryItemView.kt:43]: This will be handled in #3113

**Comment by dbrant** [comment]
> 1. When in "compare" mode, the back button should be able to cancel the "compare" mode.

Done!

> 2. Should we make the whole item togglable when in "compare" mode instead of leading the user to the detail view when clicking on the item? Feel like only making the radio button togglable is not enough.
> 3. According to the design, the first item should be yellow, and the second one should be blue.

These two items will be discussed in Design review, since there are some inconsistencies in the designs.

**Pull request merged**